src/process/drydep/ccpr_scheme_GOCART_DryDep_mod.F90

```diff
@@ -65,21 +64,19 @@ subroutine CCPr_Scheme_GOCART_DryDep(km,
       REAL, pointer, intent(in), DIMENSION(:) :: tmpu   ! Temperature [K]
       REAL, pointer, intent(in), DIMENSION(:) :: rhoa   ! Air density [kg/m^3]
       REAL, pointer, intent(in), DIMENSION(:) :: hghte  ! Height [m]
-      REAL, pointer, intent(in)                      :: radius                                ! particle radius [m]
-      REAL, pointer, intent(in)                      :: rhop                                  ! particle density [kg/m^3]
+      REAL, pointer, intent(in), optional            :: radius                                ! particle radius [m]
```

Comment:
I would move these back to required.  it isn't a problem just to pass all the variables to the scheme

Reply:
Done